src/core/src/core_logic/ExecutionConfig.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, env_layer, composite_logger, execution_parameters):
         self.sequence_number = self.__get_value_from_argv(self.execution_parameters, Constants.ARG_SEQUENCE_NUMBER)
         self.environment_settings = self.__get_decoded_json_from_argv(self.execution_parameters, Constants.ARG_ENVIRONMENT_SETTINGS)
         self.config_settings = self.__get_decoded_json_from_argv(self.execution_parameters, Constants.ARG_CONFIG_SETTINGS)
-        self.exec_auto_assess_only = bool(self.__get_value_from_argv(self.execution_parameters, Constants.ARG_AUTO_ASSESS_ONLY, False))
+        self.exec_auto_assess_only = True if (self.__get_value_from_argv(self.execution_parameters, Constants.ARG_AUTO_ASSESS_ONLY, False)).lower() == 'true' else False
```

Comment:
May be less verbose to use 
```py
(self.__get_value_from_argv(self.execution_parameters, Constants.ARG_AUTO_ASSESS_ONLY, False)).lower() == 'true'
```
instead, but I can see why the current method might be more clear. Unless there is another reason why the current method has been used?

Reply:
Updated